tutorial/source/gp.ipynb

```diff
@@ -263,14 +268,14 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "RBF$$$variance_unconstrained \n",
-      " 1.4176\n",
-      "[torch.FloatTensor of size 1]\n",
-      "\n",
       "RBF$$$lengthscale_unconstrained \n",
       " 2.7259\n",
       "[torch.FloatTensor of size 1]\n",
       "\n",
+      "RBF$$$variance_unconstrained \n",
```

Comment:
this plot doesn't seem to have converged to a reasonable result?

Reply:
Yes, because I initiate lengthscale too large (comparing to the range of our data `(0, 5)`). I just thought that it is boring to see similar plots (although we use different ways: set priors / not set priors), so I spent that section mainly to introduce the notion of constraint and how to get unconstraint value using `get_param()`. How about I just remove the plots in that section?

Reply:
alternatively you can just add a comment that the lengthscale is too large etc etc

Reply:
@martinjankowiak thanks, I just added a comment about that :)